deployment/ansible/roles.txt

```diff
@@ -1,4 +1,4 @@
-azavea.apache2,0.2.1
+azavea.apache2,0.2.2
```

Comment:
Just realized this doesn't have anything to do with this PR. The version bump adds a version glob for Apache.